### PR TITLE
[CLIPTokenizer] Improve warning

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

As can be seen in this thread the warning is a bit confusing for libraries built on top of `transformes`: https://github.com/huggingface/diffusers/issues/1388#issuecomment-1327760610

Could we maybe downgrade it to a "info" statement and remove the mentioning of BERT? 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/clip/tokenization_clip.py [hunk 1]
```diff
@@ -315,7 +315,7 @@ def __init__(
 
             self.fix_text = ftfy.fix_text
         except ImportError:
-            logger.warning("ftfy or spacy is not installed using BERT BasicTokenizer instead of ftfy.")
+            logger.info("ftfy or spacy is not installed using custom BasicTokenizer instead of ftfy.")
```
Comment (patrickvonplaten) [inline src/transformers/models/clip/tokenization_clip.py:318]: Is it really necessary to throw a warning here? I think just a logger info is good enough no? 
Also using "BERT" is a bit confusing here IMO.

**Review by LysandreJik — APPROVED** [review]
Yes, LGTM

**Review by sgugger — APPROVED** [review]
Agreed, this dates from a time where we threw warnings for everything. This is definitely just an info level.

**Comment by ydshieh** [comment]
I feel it's a bit better with a comma or even a period. 

"ftfy or spacy is not installed, using custom BasicTokenizer instead of ftfy."

I am super motivated to open a PR if you even allow me to just add a comma 😃

**Pull request merged**